src/sentry/models/relocation.py

```diff
@@ -81,6 +81,15 @@ class EmailKind(Enum):
         def get_choices(cls) -> list[tuple[int, str]]:
             return [(key.value, key.name) for key in cls]
 
+    class Provenance(Enum):
+        SELF_HOSTED = 0
+        SAAS_TO_SAAS = 1
```

Comment:
nit: technically shouldn't this be just SAAS?

Reply:
Think `SAAS_TO_SAAS` is a bit clearer and more greppable in this context.